api/src/utils.rs

```diff
@@ -0,0 +1,13 @@
+use serde_json::Value;
+use std::time::SystemTime;
+
+#[must_use]
+pub fn timestamp() -> String {
+    humantime_serde::re::humantime::format_rfc3339(SystemTime::now()).to_string()
+}
+
+pub fn map(k: impl Into<String>, v: impl Into<String>) -> serde_json::Map<String, Value> {
```

Comment:
Any place that used this function? If there's many we may convert it into macro to achieve better inlining.